docs/parameters_desc.md

```diff
@@ -0,0 +1,189 @@
+This document attempts describes the important parameters in Macrobase that
+affect its behavior.
+
+We first describe workload-specific parameters, that _need_ to be specified for
+every workload. These parameters do not have default values, so Macrobase will
+throw an exception if these aren't specified.
+
+<table class="table">
```

Comment:
I copied the table syntax from the Spark configuration file @pbailis linked me to. Agreed that this would be more consistent to the syntax used in the rest of the markdown file though.